prover/drivers/kore-driver.md

```diff
@@ -42,6 +42,12 @@ Add various standard Kore declarations to the configuration directly:
          ...
        </declarations>
 
+  rule <k> ((symbol _)  #as DECL:Declaration) => .K ... </k>
+       <declarations>
+         (.Bag => <declaration> DECL </declaration>)
+         ...
+       </declarations>
+
```

Comment:
I am not very comfortable supporting both AML and ML syntax for this simultaneously. What do we need to do to remove the ML version? Could we have `symbol S ( ARGS ) RET` desugar to the AML version, plus relevant axioms?

Reply:
We can do the desugaring, but first we need to teach `isSortOf` to work with those axioms. This will require some additional work and will probably depend on `backwardsSearch`.